### PR TITLE
Add Faker libraries

### PR DESCRIPTION
**Pull request by levonk**
Please read the CONTRIBUTING.md first. The most important parts regarding the actual entry:

- Write about it's unique selling point compared to other projects.
- If it's a commercial project, then mark it as such, e.g. `[Title ![c]](URL)`.
- Ensure that you provide concise and informative descriptions.
- Do not use a description like "A library/project/tool/framework for JSON processing in Java" since all of this is implied.
- Finish the description with a dot.
- Try to order it alphabetically.

### CODE REVIEW

**Comment by levonk** [comment]
Faking is extremely useful for testing, adding some of the libraries available for it.

**Comment by akullpp** [comment]
We already have a *Fixtures* section, how does this differ?

File: README.md [hunk 1]
```diff
@@ -340,6 +341,16 @@ A curated list of awesome Java frameworks, libraries and software.
 - [documents4j](http://documents4j.com) - API for document format conversion using third-party converters such as MS Word.
 - [docx4j](https://www.docx4java.org/trac/docx4j) - Create and manipulate Microsoft Open XML files.
 
+### Fakers
+
+*Libraries that assist with fakeing data.*
+
+- [DataFactory](https://github.com/andygibson/datafactory) - Java library for generating test data.
```
Comment (akullpp) [inline README.md]: Outdated.

File: README.md [hunk 2]
```diff
@@ -340,6 +341,16 @@ A curated list of awesome Java frameworks, libraries and software.
...
+- [faker](https://github.com/blocoio/faker) - A proguard friendly faker library, no locale support.
```
Comment (akullpp) [inline README.md]: Don't see the differenct to java-faker.

File: README.md [hunk 3]
```diff
@@ -340,6 +341,16 @@ A curated list of awesome Java frameworks, libraries and software.
...
+- [java-faker](https://github.com/DiUS/java-faker) - Port of Perl's [Data::Faker](https://metacpan.org/pod/Data::Faker) library / Ruby's [faker](https://github.com/stympy/faker) gem supports Locales.
+- [jfairy](https://github.com/Devskiller/jfairy) - Fake data generator you can try [online](https://devskiller.com/datafairy/).
+- [jpamock](https://github.com/celiovasconcelos/jpamock) - Hibernate intergrated faker.
```
Comment (akullpp) [inline README.md]: Outdated.

File: README.md [hunk 4]
```diff
@@ -340,6 +341,16 @@ A curated list of awesome Java frameworks, libraries and software.
...
 - [docx4j](https://www.docx4java.org/trac/docx4j) - Create and manipulate Microsoft Open XML files.
 
+### Fakers
+
+*Libraries that assist with fakeing data.*
+
+- [DataFactory](https://github.com/andygibson/datafactory) - Java library for generating test data.
+- [faker](https://github.com/blocoio/faker) - A proguard friendly faker library, no locale support.
+- [java-faker](https://github.com/DiUS/java-faker) - Port of Perl's [Data::Faker](https://metacpan.org/pod/Data::Faker) library / Ruby's [faker](https://github.com/stympy/faker) gem supports Locales.
+- [jfairy](https://github.com/Devskiller/jfairy) - Fake data generator you can try [online](https://devskiller.com/datafairy/).
```
Comment (akullpp) [inline README.md]: Already added.

**Comment by akullpp** [comment]
Java Faker added with https://github.com/akullpp/awesome-java/commit/cbeb78d3408f6f0e2cb9c0526125b6ac05be20d6

**Pull request closed without merging**